src/main/scala/mesosphere/marathon/core/storage/repository/impl/legacy/store/EntityStore.scala

```diff
@@ -1,10 +1,12 @@
-package mesosphere.marathon.state
+package mesosphere.marathon.core.storage.repository.impl.legacy.store
```

Comment:
Can you please also move also move all relevant marathon/state/*Test.scala classes so they live side by side?